setup.py

```diff
@@ -58,6 +58,7 @@ def get_all_extras() -> Dict:
     "valory-docker-compose==1.29.3",
     "werkzeug==2.0.3",
     "docker==6.1.2",
+    "hexbytes",
```

Comment:
Why dont we pin this version?

Reply:
Because we don't pin it to a fixed anywhere in the codebase

Reply:
https://github.com/valory-xyz/open-autonomy/blob/691fc1474659170276cf9140cceeb9e4f631f858/packages/valory/contracts/gnosis_safe/contract.yaml#L26

https://github.com/valory-xyz/open-autonomy/blob/345e9e242bbf25cc9e0f54578ee35d0db05e8a96/packages/valory/contracts/multisend/contract.yaml#L21

https://github.com/valory-xyz/open-autonomy/blob/691fc1474659170276cf9140cceeb9e4f631f858/packages/valory/skills/slashing_abci/skill.yaml#L152